odoo/addons/base/migrations/11.0.1.3/pre-migration.py

```diff
@@ -45,6 +47,121 @@ def fill_cron_action_server_pre(env):
     )
 
 
+def assure_country_state_id_on_existing_records(cr):
+    with open(get_module_resource('base', 'res', 'res.country.state.csv'),
+              newline='') as country_states_file:
+        states = csv.reader(country_states_file, delimiter=',', quotechar='"')
+        for row, state in enumerate(states):
```

Comment:
Why are you doing this? It's done when loading the CSV by regular Odoo update process

Reply:
I need each csv record to work with and ensure to solve possible conflicts that later may appear when odoo loads the file